be/tool/beck.c

```diff
@@ -969,10 +970,7 @@ static int btree(struct scanner *s, struct rectype *r, char *buf)
 		idx = ARRAY_SIZE(bt) - 1;
 
 	genadd(tree->bb_backlink.bli_gen);
-	if (!s->s_gen_found) {
-		s->s_gen_found = true;
-		s->s_gen = tree->bb_backlink.bli_gen;
-	}
+	check_and_init_genid(s, tree->bb_backlink.bli_gen);
```

Comment:
Function name  should have  verb at the end.

Reply:
Done